### DISCUSSION TITLE
can't get ladbible videos to load:

### DISCUSSION THREAD

**Discussion by computersteve** [AdguardTeam/AdGuardHome#7270] [Q&A]
I am trying to get https://www.ladbible.com/videos to load.. I have whitelisted all the domains i'm thinking are causing the problem:
mediaplex.com
analytics.google.com
ad.gt
edkt.io
audigent-gps.ybp.yahoo.com
amazon-adsystem.com
imrworldwide.com
googleads.g.doubleclick.net
id5-sync.com
jsdelivr.net
pbxai.com
rubiconproject.com
chartbeat.net
quantserve.com
hadronid.net
rlcdn.com
chartbeat.com
ad-delivery.net
permutive.com
smartadserver.com
turn.com
colossusssp.com
contextweb.com
360yield.com
1rx.io
gumgum.com
akadns.net
cm.g.doubleclick.net
pubmatic.com
adnxs.com
tapad.com
casalemedia.com
bidr.io
tpc.googlesyndication.com
ipredictive.com
stats.g.doubleclick.net
lijit.com
sonobi.com
onetag-sys.com
openx.net

Anyone have any insights or have the domain I need to actually whitelist for this to work !!! Thanks =)

**Comment by Dynamic5912** [comment]
You need to ask the list maintainers. No one here knows what blocklists you're using.
